website/app/styles/doc-components/scroll-to-top.scss

```
@@ -0,0 +1,29 @@
+@use "../typography/mixins" as *;
+
+.doc-scroll-to-top {
+  position: fixed;
+  right: 40px;
+  bottom: 40px;
+  z-index: 99;
+  width: 44px;
+  height: 44px;
+  padding: 8px 8px 4px 8px;
+  background-color: rgba(52, 53, 54, 80%);
+  border: 2px solid transparent;
+  border-radius: 50%;
+  box-shadow: 0 8px 12px 0 rgba(37, 38, 45, 8%);
+  cursor: pointer;
+  transition: bottom 150ms linear;
+
+  &:hover {
+    bottom: 43px;
+    background-color: rgba(29, 30, 31, 80%);
+    box-shadow: 0 16px 28px 0 rgba(37, 38, 45, 12%);
+  }
```

see comment above about performance
```suggestion
  &:hover {
    background-color: rgba(29, 30, 31, 80%);
    box-shadow: 0 16px 28px 0 rgba(37, 38, 45, 12%);
    transform: translate3d(0, -3px, 0);
  }
```